bot/cogs/defcon.py

```diff
@@ -252,6 +257,22 @@ def mod_log(self) -> ModLog:
                 f"**Days:** {self.days.days}"
             )
 
+        await self.update_channel_topic()
+
+    async def update_channel_topic(self):
+        """
+        Update the #defcon channel topic with the current DEFCON status
+        """
+
+        if self.enabled:
+            day_str = "days" if self.days > 1 else "day"
+            new_topic = f"{BASE_CHANNEL_TOPIC}\n(Status: Enabled, Threshold: {self.days} {day_str})"
```

Comment:
Delta cast to string show as `1 day, 0:00:00` instead of `1`.
```suggestion
             new_topic = f"{BASE_CHANNEL_TOPIC}\n(Status: Enabled, Threshold: {self.days.days} {day_str})"
```